pegasus/sites.v3/advocacy.code.org/public/index.haml

```diff
@@ -18,9 +18,11 @@ video_player: false
 - banner_body = capture_haml do
   Bringing together 100 industry, nonprofit, and advocacy organizations, the Code.org Advocacy Coalition is growing the movement to make computer science a fundamental part of K-12 education.
 
-  .join-now
-    %a.link-button{href: "https://bit.ly/joinadvocacycoalition", target: "_blank"}
+  .button-wrapper
+    %a.link-button.has-external-link{href: "https://bit.ly/joinadvocacycoalition", target: "_blank", rel: "noopener noreferrer"}
```

Comment:
For my own learning: what does .has-external-link do?

Reply:
This class adds the little box arrow icon next to buttons and text links that open in a new tab 😁https://github.com/code-dot-org/code-dot-org/blob/f76661a133b2661fe8ef55463e7b4fcd308d2ec4/pegasus/sites.v3/code.org/styles_min/040-page.css#L344-L358